.github/workflows/clean_up_package_registry.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+name: Clean up packages in registry
```

Comment:
Suggest: Clean up packages in GitHub packages registry

Reply:
Nope, the action can pass a different registry. I should change in other parts

Reply:
Which now i see i'm wrong, since the url registry is only github. You can change the registry for Github enterprise, but still github.
Fixing

Reply:
done